app/config.ts

```diff
@@ -0,0 +1,5 @@
+function getEnvValue<T>(obj: any, key: string): T {
```

Comment:
I think this generic type is not needed here - we will always return string currently. Sorry for confusion.